package.json

```diff
@@ -57,15 +57,15 @@
     "@artsy/auto-config": "1.0.1",
     "@artsy/palette": "5.1.5",
     "@babel/cli": "7.0.0",
-    "@babel/core": "7.3.4",
+    "@babel/core": "^7.7.2",
     "@babel/plugin-proposal-class-properties": "7.3.4",
     "@babel/plugin-proposal-decorators": "7.3.0",
     "@babel/plugin-proposal-json-strings": "7.2.0",
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
     "@babel/plugin-syntax-import-meta": "7.0.0",
     "@babel/preset-env": "7.3.4",
     "@babel/preset-react": "7.0.0",
-    "@babel/preset-typescript": "7.3.3",
+    "@babel/preset-typescript": "^7.7.2",
```

Comment:
Can you remind me why this was needed?

Reply:
I see, so it’s because of syntax needed in stories.

Reply:
`as const`